monolith/views/stories.py

```diff
@@ -50,3 +56,22 @@ def _like(authorid, storyid):
     else:
         message = 'You\'ve already liked this story!'
     return _stories(message)
+
+# TODO to complete
+@stories.route('/stories/new_story', methods=['GET'])
```

Comment:
This should have a POST method which creates the unpublished story in the database and then redirect to /write_story [GET], right?

I am asking to make sure that we are on the same page.